portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,11 +38,28 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String currComment = request.getParameter("comment");
-    writeComments(currComment);
-    String json = convertToJson(readComments());
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
+    String translatedText = translateComment(request);
+    if(translatedText != "") {
```

Comment:
In java you can do if (!translatedText.isEmpty()) {}

Reply:
done